src/main/java/uk/gov/hmcts/ccd/domain/service/UserProfileOperation.java

```diff
@@ -84,8 +84,9 @@ private boolean areDefaultsEqual(final UserProfile userProfileFromRepository,
                                   userProfileFromDefintion.getWorkBasketDefaultState())// checkstyle line break
             && userProfileFromRepository.getJurisdictions()
             .stream()
-            .filter(j -> StringUtils.equals(j.getId(), userProfileFromDefintion.getWorkBasketDefaultJurisdiction()))
-            .count() > 0;
+            .anyMatch(j -> StringUtils.equals(
+                j.getId(),
+                userProfileFromDefintion.getWorkBasketDefaultJurisdiction()));
```

Comment:
What was this change for?

Reply:
In IntelliJ IDE there is a refactoring tool that automatically helps you detect and simplify complex code. Have a try.